examples/multiview_tfm_non_line_of_sight/multiview_tfm_non_line_of_sight.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
```

Comment:
Neat new example! Out of curiosity, is there a flaw in the examination object?

Reply:
Yes, this is a data set used in [my paper](https://doi.org/10.1016/j.ndteint.2023.103014) which has a SDH in the NLOS region. Will make the plots a little nicer by adding in a point at the defect location, and also add in the interfaces to make it a bit clearer where the geometry is.